extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/Common.java

```diff
@@ -43,7 +43,7 @@ static SpanContext buildSpanContext(String traceId, String spanId, String sample
           spanId,
           traceFlags,
           TraceState.getDefault());
-    } catch (Exception e) {
+    } catch (RuntimeException e) {
```

Comment:
New errorprone doesn't like missing checked exceptiosn in a catch Exception block